internal/operators/cnv/manifest.go

```diff
@@ -140,3 +144,34 @@ metadata:
   namespace: "{{.OPERATOR_NAMESPACE}}"
 spec:
   BareMetalPlatform: true`
+
+const cnvHPPManifest = `apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+    spec:
+      imagePullPolicy: IfNotPresent
+      storagePools:
+        - name: sno
+    pvcTemplate:
+      storageClassName: localblock-sc
+      volumeMode: Block
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 50Gi
```

Comment:
What this number means?
What will happen if there is a disk with less space?

Reply:
This is the capacity of the pool that the HPP can use to dynamically provision storage on the host's path.
We could work with less, but that would mean less persistent VMs running alongside each other, 50Gi seems fair.

Are you suggesting we derive this value from discovery? any tips on approaching that?

Reply:
The same comments from the original pr. if we have minimal requirement it should be validated.

Reply:
https://github.com/openshift/assisted-service/pull/3072#discussion_r771971942